certified-connectors/OpenExperience/apiDefinition.swagger.json

```diff
@@ -0,0 +1,249 @@
+                      "description": "Set to \"Yes\" if you wan't to allow Project Administrators to be able to edit subcontractors in this project.",
+                      "title": "Can Project Admin edit subcontractors?",
+                      "enum": [
+                        true,
+                        false
+                      ],
+                      "default": false
+                    }
+                  },
+                  "description": "settings",
+                  "required": [
+                    "projectAdminMembersAccess"
+                  ]
+                }
+              },
+              "required": [
+                "customerId",
+                "services",
+                "contactEmail",
+                "contactPhone",
+                "name",
+                "id",
+                "responsible",
+                "settings"
+              ]
+            }
+          }
+        ],
+        "description":"Creates new project in Open Experience for selected services"
+      }
+    },
+    "/connector/getUserCustomers": {
+      "get": {
+        "responses": {
+          "200": {
+            "description": "default",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "accounts": {
+                  "type": "array",
+                  "items": {
+                    "type": "object",
+                    "properties": {
+                      "id": {
+                        "type": "string",
+                        "description": "id"
+                      },
+                      "label": {
+                        "type": "string",
+                        "description": "label"
+                      },
+                      "sortindex": {
+                        "type": "integer",
+                        "format": "int32",
+                        "description": "sortindex"
+                      }
+                    }
+                  },
+                  "description": "accounts"
+                }
+              }
+            }
+          }
+        },
+        "summary": "Get Available Customers",
+        "operationId": "GetAvailableCustomers",
+        "x-ms-visibility": "internal",
+        "parameters": []
+      }
+    },
+    "/connector/getUserServices": {
+      "get": {
+        "responses": {
+          "200": {
+            "description": "defaul1t",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "services": {
+                  "type": "array",
+                  "items": {
+                    "type": "object",
+                    "properties": {
+                      "type": {
+                        "type": "string",
+                        "description": "type"
+                      },
+                      "serviceFullname": {
+                        "type": "string",
+                        "description": "serviceFullname"
```

Comment:
@oxTomaszJ All the description values are expected to be free from grammatical and spell errors. Please provide valid description values for all the properties.